primitives/src/policy.rs

```diff
@@ -199,7 +199,7 @@ impl Policy {
     pub fn epoch_index_at(block_number: u32) -> u32 {
         // Any block before the genesis is considered to be part of epoch 0
         if block_number < Self::genesis_block_number() {
-            panic!("There are no epochs prior to the genesis block");
+            block_number
```

Comment:
This should probably return `0` instead of the input block number? The comment says anything before the genesis block is epoch 0.

Reply:
Not really, the epoch index is a bad name for what it is: _the epoch index is the number of a block relative to the epoch it is in_